specs/extension_image.md

```diff
@@ -34,9 +34,103 @@ sysext images extend `/usr/` (OS vendor tree) and/or `/opt/` (third-party vendor
 confext images extend `/etc`. They must contain a `/etc/extension-release.d/extension-release.<IMAGE>` file
 to identify them.
 
-## Image Content
-Extension Images should be additive, and not override content present in the base image or other DDIs,
-but this will not be enforced.
+<h2 id="image-content">Image Content</h2>
+Extension Images should be additive, and not override content present in the base image or other DDIs.
+However, there currently is no safe and efficient way to detect collisions and to enforce content uniqueness
+across the stack of images.
+In a future version of this specification options for enforcing uniqueness may be provided.
+
+## Base Directory Immutability / Mutability
+By default, the base directory (`/etc`, `/usr`, `/opt`) overlaid by an extension is rendered immutable
```

Comment:
in systemd we have this rule that paths in docs and log msgs that reference a dir should always be suffixed with `/` to make this clear. I think it would make sense to follow the same rule here?

Reply:
I've updated the PR - makes sense to better distinguish between files and directories.